### PR TITLE
ProctorTrack changes

### PR DESCRIPTION
**Pull request by shaidar**
#### What's this PR do?
ProctorTrack changes that have been manually deployed and tested on QA

### CODE REVIEW

File: pillar/edx/ansible_vars/residential.sls [hunk 1]
```diff
@@ -167,6 +167,30 @@ edx:
...
+      PROCTORING_SETTINGS:
+        MUST_BE_VERIFIED_TRACK: false
+      PROCTORING_BACKENDS:
+        DEFAULT: "proctortrack"
+        proctortrack:
+          client_id: __vault__::secret-{{ business_unit }}/{{ environment }}/proctortrack>data>client_id
+          client_secret: __vault__::secret-{{ business_unit }}/{{ environment }}/proctortrack>data>client_secret
+          base_url: __vault__::secret-{{ business_unit }}/{{ environment }}/proctortrack>data>base_url
+      JWT_AUTH:
+        JWT_AUDIENCE: 'pt_openedx_key'
```
Comment (blarghmatey) [inline pillar/edx/ansible_vars/residential.sls:179]: This seems like a duplicate of lines 182 and 183

File: pillar/edx/ansible_vars/residential.sls [hunk 2]
```diff
@@ -167,6 +167,30 @@ edx:
...
+        AUDIENCE: "https://{{ purpose_data.domains.lms }}"
+      JWT_AUTH:
+        JWT_AUDIENCE: "pt_openedx_key"
+        JWT_ISSUER: "https://{{ purpose_data.domains.lms }}/oauth2"
+        JWT_ISSUERS:
+          - AUDIENCE: "pt_openedx_key"
+          - ISSUER: "https://{{ purpose_data.domains.lms }}/oauth2"
+          - SECRET_KEY: __vault__::secret-{{ business_unit }}/{{ environment }}/jwt_secret_key>data>value
+        JWT_SECRET_KEY: __vault__::secret-{{ business_unit }}/{{ environment }}/jwt_secret_key>data>value
+        JWT_PUBLIC_SIGNING_JWK_SET: null
```
Comment (blarghmatey) [inline pillar/edx/ansible_vars/residential.sls:190]: Why are we setting these to null?

Comment (blarghmatey) [reply pillar/edx/ansible_vars/residential.sls:190]: Don't we already have the JWK defined elsewhere in the config?

File: pillar/edx/ansible_vars/residential.sls [hunk 3]
```diff
@@ -167,6 +167,30 @@ edx:
...
+          base_url: __vault__::secret-{{ business_unit }}/{{ environment }}/proctortrack>data>base_url
+      JWT_AUTH:
+        JWT_AUDIENCE: 'pt_openedx_key'
+      DEFAULT_JWT_ISSUER:
+        AUDIENCE: "https://{{ purpose_data.domains.lms }}"
+      JWT_AUTH:
+        JWT_AUDIENCE: "pt_openedx_key"
+        JWT_ISSUER: "https://{{ purpose_data.domains.lms }}/oauth2"
+        JWT_ISSUERS:
+          - AUDIENCE: "pt_openedx_key"
```
Comment (blarghmatey) [inline pillar/edx/ansible_vars/residential.sls:186]: Should these all be separate elements in a list, or is this a list of dicts?

File: pillar/edx/ansible_vars/residential.sls (hunk 1 above)
Comment (shaidar) [reply pillar/edx/ansible_vars/residential.sls:179]: Removed

File: pillar/edx/ansible_vars/residential.sls (hunk 2 above)
Comment (shaidar) [reply pillar/edx/ansible_vars/residential.sls:190]: Fixed

File: pillar/edx/ansible_vars/residential.sls (hunk 3 above)
Comment (shaidar) [reply pillar/edx/ansible_vars/residential.sls:186]: This one's weird, cause that's how it is in the `lms.yml` file so guess when salt renders it's gonna remove the hyphen

**Comment by blarghmatey** [comment]
:+1:

File: pillar/edx/ansible_vars/residential.sls (hunk 3 above)
Comment (shaidar) [reply pillar/edx/ansible_vars/residential.sls:184]: That's how it was setup on QA before

**Pull request merged**